src/Sfa.Roatp.Indexer.ApplicationServices/ProviderIndexer.cs

```diff
@@ -144,15 +144,10 @@ public IEnumerable<RoatpProvider> LoadEntries()
             return _providerDataService.GetRoatpData();
         }
 
-        public async Task IndexEntries(string indexName, IEnumerable<RoatpProvider> roatpProviders)
+        public void IndexEntries(string indexName, IEnumerable<RoatpProvider> roatpProviders)
         {
-            var bulkApiProviderTasks = new List<Task<IBulkResponse>>();
-            roatpProviders = roatpProviders.ToList();
-
             _log.Debug("Indexing " + roatpProviders.Count() + " RoATP providers");
-            bulkApiProviderTasks.AddRange(_indexMaintainer.IndexRoatpProviders(indexName, roatpProviders));
-
-            _indexMaintainer.LogResponse(await Task.WhenAll(bulkApiProviderTasks), "RoatpProviderDocument");
+            _indexMaintainer.IndexRoatpProviders(indexName, roatpProviders.ToList());
```

Comment:
Needs a test?

Reply:
same as my previous comment - the calling method IndexerService.CheckRoatpAndCreateIndexAndUpdateAlias has already converted the IEnumerable to a list